requirements.txt

```diff
@@ -1,10 +1,10 @@
 Django>=3.2,<4.2
 wagtail>=2.15,<5.0
-wagtailmedia>=0.13
 graphene-django>=3.0.0
 factory-boy==3.2.1
 wagtail-factories>=4.0.0,<4.1.0
 django-cors-headers==3.4.0
+wagtailmedia>=0.13
 wagtail-headless-preview
 # simplified database configuration to support testing multiple DB engines in CI.
 dj-database-url==1.2.0
```

Comment:
this and `requirements-channels.txt` are test requirements. I suggest moving them to `tests/`

Reply:
There are some dependencies for the testapp, but we do import from wagtail and django in grapple/*, should I keep those in the /requirements[-channels].txt or just move everything to the testapp? I was kind of thinking to defer altering requirements to the flit pr to trim some scope here since I'm not testing packaging I didn't want to have any unintended impacts.

Reply:
The requirements*.txt files are only used for tests. well, outside of this PR, in the `example` app

The imports under grapple/* should be all handled via the install requirements (i.e. https://github.com/torchbox/wagtail-grapple/blob/main/setup.cfg#L42-L53)

Reply:
done.